### PR TITLE
Fixed some Grammar mistakes in install directory

### PR DESCRIPTION
**Pull request by mohantym**
Fixed some Grammar mistakes in source.md, source_windows.md and pip.md which were found using inspect code feature of Pycharm.

### CODE REVIEW

File: site/en/install/source.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ Found cuDNN 7 in:
 
 
 Please specify a list of comma-separated CUDA compute capabilities you want to build with.
-You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus. Each capability can be specified as "x.y" or "compute_xy" to include both virtual and binary GPU code, or as "sm_xy" to only include the binary code.
+You can find compute capability of your device at: https://developer.nvidia.com/cuda-gpus. Each capability can be specified as "x.y" or "compute_xy" to include both virtual and binary GPU code, or as "sm_xy" to only include the binary code.
```
Comment (8bitmp3) [inline site/en/install/source.md:144]: Please review software-suggested changes and take into account things like context. Reversing this change:

```suggestion
You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus. Each capability can be specified as "x.y" or "compute_xy" to include both virtual and binary GPU code, or as "sm_xy" to only include the binary code.
```

File: site/en/install/source_windows.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ Please specify the cuDNN version you want to use. [Leave empty to default to cuD
 Please specify the location where cuDNN 7 library is installed. Refer to README.md for more details. [Default is C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v9.0]: <b>C:\tools\cuda</b>
 
 Please specify a list of comma-separated Cuda compute capabilities you want to build with.
-You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
+You can find compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
```
Comment (8bitmp3) [inline site/en/install/source_windows.md:134]: Please review software-suggested changes and take into account things like context. Reversing this change:

```suggestion
You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
```

**Review by 8bitmp3 — APPROVED** [review]
Thanks for some of the fixes, appreciate it. Please review software-suggested changes and verify them before submitting them next time, as not all changes may make sense.

**Comment by mohantym** [comment]
@8bitmp3 !
Thanks for the review. Yes!  will test the build internally through bazel before raising further PR's ( same suggested by @gbaned today)
Thank you!

**Pull request merged**